app/poros/github_repo.rb

```diff
@@ -0,0 +1,8 @@
+class GithubRepo
+
+   def self.name
+     github_data = GithubService.repo_info
+     github_data[:name]
```

Comment:
same deal here -- I don't think we need the variable and I find it to be more clutter than clarity